website/docs/type-assertions.md

```diff
@@ -8,7 +8,7 @@ There are five ways to assert the types of expressions in Sorbet:
 
 - `T.let(expr, Type)`
 - `T.cast(expr, Type)`
-- `T.must(expr)`
+- `T.must(expr)` / `T.must_because(expr, msg)`
```

Comment:
Nit: `T.must_because` takes the `msg` as a block, not a param...though I don't know how to succinctly represent that here.